### DISCUSSION TITLE
Why do you have "templates" displayed, when you can't use them anywhere?

### DISCUSSION THREAD

**Discussion by demetre19** [webstudio-is/webstudio-community#64] [Q&A] [answered]
Probably better to call them examples.
Unless I am wrong, and they are accessible, but they are certainly not obvious, as a first time user.

I think more wireframe pages will do wonders for your customer base.

**Comment by kof** [comment]
I am confused, what do you mean you can't use them? what happens when you click on them?

**Comment by demetre19** [comment]
<img width="1763" alt="f68436a15817361830fc60787cb4dfb97e4283c912f09191bf02aa9ae846a07b" src="https://github.com/webstudio-is/webstudio-community/assets/29856708/af56ebb7-9886-4df9-af9f-df87de6f015d">

**Reply by kof** [reply]
I think you answer with an image by email and github didn't attach it

**Reply by itsNintu** [reply]
Reupload of the image. ![277464124-af56ebb7-9886-4df9-af9f-df87de6f015d.png](https://github.com/webstudio-is/webstudio-community/assets/128840440/cc962258-5a6a-4d01-bdef-3fc623c32536)

**Comment by ghost** [comment, marked as answer]
No wonder @demetre19 is confused - this is a really unusual link pattern. Idk how this happened but it should not be this way. With any card-style link, the whole card should be one link unless there are clear buttons. It's not intuitive at all that clicking on the image will take you to a different place than clicking on the bottom area.

We need to fix this by making the whole card open the new project when clicked. I am making a github issue for this fix.

**Reply by ghost** [reply]
[#2601](https://github.com/webstudio-is/webstudio/issues/2601)

**Comment by ghost** [comment]
We will improve these cards by giving them a clearly separate link to open the published page.

Also as @itsNintu suggested we should indicate that opening a template means adding it to your projects. Maybe it would be best if the user could open the project in Builder view and can clone it to their projects from there.
